solutions/my_select_solution.rb

```diff
@@ -1,4 +1,8 @@
 class Array
+  # define a method `my_select(&prc)` that correctly returns
```

Comment:
Should be `Array#my_select` or otherwise specify that it's monkey patched onto the Array class